src/SwedbankPay.Sdk/PaymentInstruments/MobilePay/MobilePayPaymentRequest.cs

```diff
@@ -1,91 +1,91 @@
-                                                    string payerReference,
-                                                    string userAgent,
-                                                    CultureInfo language,
-                                                    Urls urls,
-                                                    PayeeInfo payeeInfo,
-                                                    Dictionary<string, object> metadata = null,
-                                                    PrefillInfo prefillInfo = null)
-            {
-                Operation = operation ?? throw new ArgumentNullException(nameof(operation));
-                Intent = intent;
-                Currency = currency;
-                Prices = prices;
-                Description = description;
-                PayerReference = payerReference;
-                UserAgent = userAgent;
-                Language = language;
-                Urls = urls;
-                PayeeInfo = payeeInfo;
-                Metadata = metadata;
-                PrefillInfo = prefillInfo;
-            }
-
-
-            public CurrencyCode Currency { get; set; }
-            public string Description { get; set; }
-            public Intent Intent { get; set; }
-            public CultureInfo Language { get; set; }
-            public Dictionary<string, object> Metadata { get; }
-            public Operation Operation { get; set; }
-            public PayeeInfo PayeeInfo { get; internal set; }
-            public string PayerReference { get; set; }
-            public List<Price> Prices { get; set; }
-            public Urls Urls { get; }
-            public string UserAgent { get; set; }
-            public PrefillInfo PrefillInfo { get; set; }
-        }
-
-
-        public class MobilePayPaymentRequestObject
-        {
-            protected internal MobilePayPaymentRequestObject(Uri shoplogoUrl)
-            {
-                ShoplogoUrl = shoplogoUrl;
-            }
-
-            /// <summary>
-            ///    URI to logo that will be visible at MobilePay payment menu
-            /// </summary>
-            public Uri ShoplogoUrl { get; set; }
-
-        }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.Payments.MobilePayPayments
+{
+    public class MobilePayPaymentRequest
+    {
+        public MobilePayPaymentRequest(Operation operation,
+                              Intent intent,
+                              CurrencyCode currency,
+                              List<Price> prices,
+                              string description,
+                              string userAgent,
+                              CultureInfo language,
+                              Urls urls,
+                              PayeeInfo payeeInfo,
+                              Uri shopslogoUrl,
+                              string payerReference = null,

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)